nss-day-cohort-48/the-worst-site-ever-eliminatorz
# Created a rough draft for the homepage

Added a basic navbar and 4 intro sections for the 4 of us as well as a crude background for the page and gradient backgrounds for the intro sections and navbar.

STEPS TO TEST:

1. Pull my code
2. Serve locally
3. Verify the homepage is ugly on purpose
